### PR TITLE
test(ui): Adding tests for the cancel request workflow

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR adds some tests for the cancel request workflow

1. `should be able to cancel a request if the user is the requester`
2. `should be able to see how many CVEs will be affected by a cancel`

### CODE REVIEW

File: ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/cancelRequestFlow.test.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    beforeEach(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
```
Comment (vjwilson) [inline ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/cancelRequestFlow.test.ts:41]: Would it be would making this 2nd beforeEach block just an else, or fall-through block of a single beforeEach block?

**Review by vjwilson — COMMENTED** [review]
One question (suggestion)

File: ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/cancelRequestFlow.test.ts (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/cancelRequestFlow.test.ts:41]: Ah, you're suggesting merging the two `beforeEach` into one. That would be reasonable to do. I see an issue where the tests are not properly canceling the exceptions. Maybe there's a race condition involved with these two `beforeEach` functions. Merging them might resolve that issue as well.

**Review by dvail — APPROVED** [review]
LGTM 👍 

Worthwhile to rerun the failing e2e jobs just to be sure they are normal CI flakes.

The component-test failure is something I see occasionally too, and am keeping an eye on.

**Pull request merged**